packages/react-styled-core/src/ToggleSwitch/index.js

```diff
@@ -0,0 +1,107 @@
+import { forwardRef } from 'react';
+import PseudoBox from '../PseudoBox';
+import Box from '../Box';
+import ControlBox from '../ControlBox';
+import VisuallyHidden from '../VisuallyHidden';
+import useToggleSwitchStyle from './styles';
+
+const ToggleSwitch = forwardRef(
+  (
+    {
+      id,
+      name,
+      value,
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+      variantColor,
+      defaultIsChecked,
+      isChecked,
+      size,
+      isDisabled,
```

Comment:
Recommended naming convention for Button, Input, Checkbox, Radio:
* `defaultChecked`
* `checked`
* `disabled`
* `readOnly`

Reply:
fixed in 2ce388a